src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -175,10 +212,18 @@ public DBUser getUserByName(final String userName) {
       ps.close();
       return user;
     } catch (final SQLException sqle) {
-      s_logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
+      logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
       throw new IllegalStateException(sqle.getMessage());
     } finally {
       DbUtil.closeConnection(con);
     }
   }
+
+  private static String mask(final HashedPassword hashedPassword) {
```

Comment:
nit: do not dump private static methods at teh bottom of the class file, minimize vertical distance as described in the dev docs. Eclipse may have a setting for this to do automatically.